prereise/gather/griddata/transmission/helpers.py

```diff
@@ -48,3 +63,27 @@ def translate_to_per_unit(x, nominal_unit, z_base):
     if nominal_unit.lower() in {"siemen", "siemens"}:
         return x * z_base
     raise ValueError(f"Unknown nominal unit: {nominal_unit}")
+
+
+def get_standard_conductors():
+    """Read the data file on standard conductor parameter values.
+
+    :return: (*pandas.DataFrame*) -- data frame, indexed by conductor code name.
+    """
+    filepath = os.path.join(os.path.dirname(__file__), "data", "conductors.csv")
+    return pd.read_csv(filepath, index_col=0)
+
+
+def approximate_loadability(length_km, method="power"):
+    """Approximate the value of the St. Clair curve at a given point, using a given
+    approximation method.
+
+    :param float length_km: line length (kilometers).
+    :param str method: curve approximation method. Currently, only 'power' is supported.
+    :return: (*float*) -- line loadibility (normalized to surge impedance loading).
+    :raises ValueError: if the ``method`` is not supported.
+    """
+    miles = length_km / miles_per_kilometer
+    if method == "power":
+        return st_clair_power_approx_coeff * miles**st_clair_power_approx_exponent
+    raise ValueError(f"Unsupported method: {method}")
```

Comment:
Should the error message list the available methods?

Reply:
Done.